src/store/resources/resources-actions.ts

```diff
@@ -20,12 +20,14 @@ export const updateResources = (resources: Resource[]) => resourcesActions.SET_R
 
 export const loadResource = (uuid: string) =>
     async (dispatch: Dispatch, getState: () => RootState, services: ServiceRepository) => {
-        const kind = extractUuidKind(uuid);
-        const service = getResourceService(kind)(services);
-        if (service) {
-            const resource = await service.get(uuid);
-            dispatch<any>(updateResources([resource]));
-            return resource;
-        }
+        try {
+            const kind = extractUuidKind(uuid);
+            const service = getResourceService(kind)(services);
+            if (service) {
+                const resource = await service.get(uuid);
+                dispatch<any>(updateResources([resource]));
+                return resource;
+            }
+        } catch {}
```

Comment:
We could add something in catch section that informs about an exception. For example breadcrumb with 'Resource could not be loaded' message.

Reply:
I'm not sure if we really want that as this function is a part of bigger workflows and IMO it should fail silently.